extension/src/shared/reearth/layers/mvt.ts

```diff
@@ -107,11 +118,18 @@ export const MVTLayer: FC<MVTProps> = ({ url, onLoad, visible, appearances, laye
     }),
     [url, layers],
   );
-
+  const preUpdateOrder = useRef<number>();
   const handleOnLoad = useCallback(
     (layerId: string) => {
       if (!meta) return;
 
+      if (layerId) {
+        if (smallIndexMvtLayer?.layerId) {
+          window.reearth?.layers?.bringToFront?.(smallIndexMvtLayer.layerId);
+        }
+      }
```

Comment:
If possible, this should be handled in useEffect. Because handleOnLoad is executed after 300ms in useLayer, so it might be slow.